src/aequitas/bias.py

```diff
@@ -1,7 +1,12 @@
 import logging
 from sys import exit
 
+from aequitas.preprocessing import get_attr_cols
+from aequitas.plotting import Plot
+assemble_ref_groups = Plot._assemble_ref_groups
```

Comment:
What's the logic behind this?

* Is this a useful helper, only loosely associated with `Plot`? In that case, define it as a module-level function in `plotting`.
* Is it a `Plot` method of external utility (as it appears to be)? Then no need to "hide" it with a leading underscore.